### PR TITLE
changed to api 4

### PR DESCRIPTION
**Pull request by cristinazhao-toast**
updated the Looker API to 4 with Kelly

QA:
Screenshot 2023-04-13 at 10 29 46 AM

### CODE REVIEW

**Comment by alisa-aylward-toast** [comment]
since this is a pypi package, you need to cut a new release @cristinazhao-toast! this is fun, trust me.
can you:
* increase the version in setup.py
* fill out info in changelog.md

**Comment by alisa-aylward-toast** [comment]
@cristinazhao-toast  when i say fill out info i mean add this:

# Changelog

## [1.X.0] - 2023-XX-XX

- What you changed

**Comment by cristinazhao-toast** [comment]
lmk if you prefer 1.3.0 instead

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.2.1] - 2023-04-13
```
Comment (alisa-aylward-toast) [inline CHANGELOG.md]: maybe this is major release? 1.3?

Comment (alisa-aylward-toast) [reply CHANGELOG.md]: 1.3 is minor actually! 2.0 is major! let's just do 1.3

**Pull request merged**